pixart.js

```diff
@@ -1 +1,98 @@
+ $( document ).ready( function(){
 
+//---------------- Sets up the page
+
+   //Sets default brush box value
+   $("#color-field").val("#DB200A");
+
+   //Appends canvas squares to body on load
+   for(i = 0; i < 18000; i++){
+     $(".drawcanvas").append("<div class=\"square\"></div>");
+   }
+
+//---------------- Color selector
+
+   //Fills brush box bg with hex color from input on Enter keydown
+   //@@@@ TODO: Fix this.
+   $("color-field").keydown(function(){
+      if(e.keyCode == 13){
+        event.preventDefault();
+        $(".brush").css("background-color", $("#color-field").val());
+        console.log("the enter key was pressed");
+      }
+    });
+
+    //Fills brush box bg with hex color from input on button click
+    $("#set-color").click(function(){
+      event.preventDefault();
+      $(".brush1").css("background-color", $("#color-field").val());
+      console.log("a click occurred");
+    });
+
+    //Fills brush box bg with red on red box click
+    $(".brush2").click(function(){
+      event.preventDefault();
+      $(".brush1").css("background-color", "#DB200A");
+      $("#color-field").val("#DB200A");
+    });
+
+    //Fills brush box bg with blue on blue box click
+    $(".brush3").click(function(){
+      event.preventDefault();
+      $(".brush1").css("background-color", "#FEF8FF");
+      $("#color-field").val("#FEF8FF");
+    });
```

Comment:
I see a lot of duplication through these event handlers too.  Maybe a function that you pass a color too?  Removing duplication makes it much easier to maintain.
```
function setColor(color){
  $(".brush1").css("background-color", color);
  $("#color-field").val(color);
}
```